enacts/onset/layout.py

```diff
@@ -5,7 +5,7 @@
 from dash import dash_table
 import dash_leaflet as dlf
 import plotly.express as px
-from .widgets import Block, Sentence, Date, Units, Number
+from .components import Block, Sentence, Date, Units, Number
```

Comment:
uses DateNoYear so import that instead of Date that is not used.
Also Units is not used so could be removed from the imports